crates/genesis/src/mainnet.genesis.json

```diff
@@ -66,7 +66,7 @@
     "depositContractAddress": null
   },
   "nonce": "0x0",
-  "timestamp": "0x0",
+  "timestamp": "0x690B792F",
```

Comment:
nit: lowercase is convention for ethereum number

Reply:
https://github.com/init4tech/node-components/pull/48/commits/5ed762471555d69411dd499b441c78e6f1fa5433